EventPlanning.service/Controllers/EventsController.cs

```diff
@@ -90,6 +97,9 @@ public List<DAL.Activity> GetEventActivities(int eventId)
         [HttpPost]
         public IActionResult CreateEvent([FromBody]EventData eventData)
         {
+            Int32.TryParse(User.FindFirstValue("jti"), out var id);
```

Comment:
Maybe we can have this code in a static method somewhere? Utils project? Fully tested and so on? (That will expect null values and w/e